<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html>
<head>
  <title>TooT Suite Project</title>
  <link rel="stylesheet" href="tootsuite.css">
</head>

<body>

<!-- Main content -->
<header>
<h1>TooT Suite: Prediction and classification of membrane transport proteins</h1>
<h2>A Genome Canada 2017 Bioinformatics and Computational Biology Competition Project</h2>
</header>

<nav>
<ul class="navbar">
<li><a href="index.html">Home</a></li>
<li><a href="https://www.concordia.ca/faculty/gregory-butler.html" target="_BLANK">Gregory Butler</a></li>
<li><a href="https://users.encs.concordia.ca/~tglatard/" target="_BLANK">Tristan Glatard</a></li>
<li><a href="https://github.com/orgs/bioinformatics-group/repositories?type=all" target="_BLANK">Software</a></li>
<li><a href="datasets.html">Datasets</a></li>
<li><a href="https://github.com/boutiques/boutiques" target="_BLANK">Boutiques</a></li>
<li><a href="https://www.genomecanada.ca/en/toot-suite-predication-and-classification-membrane-transport-proteins" target="_BLANK">Genome Canada Description</a></li>
</nav>

<section>
The Toot Suite project is developing tools for predicting and classifying transport proteins
in host organisms and microbiomes.
<p>
These tools will provide scientists with insight into host-microbiome interactions.
For agricultural scientists, the host-microbiome interactions are important for plant and animal health and growth,
which are key factors in food productivity.
<h2>An Open Science Project following F.A.I.R. Guidelines</h2>
A key aspect of the TooT Suite project is to support open, reproducible science.
Therefore, we adopt the F.A.I.R. Guidelines for Findable, Accessible, Interoperable, and Reusable project artefacts.
This includes our experimental platform based on
<i>Boutiques</i>
for computation on the clusters of Compute Canada and the Gina Cody School of Engineering and Computer Science.
and a git repository holding our datasets, source code, and processing pipelines.
<h2>Deliverables</h2>
<table style="width:100%">
  <tr>
    <th>Due</th>
    <th>Deliverable</th> 
    <th>Description</th>
    <th>Link</th>
  </tr>
  <tr>
    <td>Q4</td>
    <td><b>D.1.1</b></td> 
    <td><i>TooT-SC</i>: Given a fasta file of protein sequences, predict the substrate class that is transported.<br/>
    <i>TooT-BERT-SC</i>: Classifies transmembrane transporter proteins by substrate using a majority-vote approach with various standard predictors.</td>
    <td><a href="https://github.com/bioinformatics-group/TooT-SC/"  target="_BLANK">Link</a><br/>
    <a href="https://github.com/bioinformatics-group/TooT-BERT-SC"  target="_BLANK">Link</a></td>
  </tr>
  <tr>
    <td>Q4</td>
    <td><b>D.1.4</b></td> 
    <td>Computational infrastructure for <i>in silico</i> experiments.</td>
    <td><a href="https://github.com/bioinformatics-group/bioinformatics-singularity"  target="_BLANK">Link</a></td>
  </tr>
  <tr>
    <td>Q4</td>
    <td><b>D.4.1</b></td> 
    <td>Experimental infrastructure.</td>
    <td><a href="https://github.com/boutiques/boutiques"  target="_BLANK">Link</a></td>
  </tr>
  <tr>
    <td>Q6</td>
    <td><b>D.1.2</b></td> 
    <td><i>TooT-Proteome</i>: Given a fasta file of a proteome, predict the transporters and their properties.<br/>
    <i>TooT-BERT-T</i>: Predicts transmembrane transporter proteins from protein sequences using ProtBERT-BFD and logistic regression.<br/>
    <i>TooT-Proteome</i>: Predicts transmembrane transporter proteins and their substrate-specific properties using BERT-based predictors for transporter and substrate classification.</td>
    <td><a href="https://github.com/bioinformatics-group/TooT-Proteome"  target="_BLANK">Link</a><br/>
    <a href="https://github.com/bioinformatics-group/TooT-BERT-T"  target="_BLANK">Link</a><br/><br/>
    <a href="https://github.com/bioinformatics-group/TooT-BERT-Proteome"  target="_BLANK">Link</a></td>
    
  </tr>
  <tr>
    <td>Q6</td>
    <td><b>D.2.1 and D.3.1 (HPC)</b></td> 
    <td><i>TooT-TC</i>: Given a fasta file of protein sequences, predict their TC family and subfamily.</td>
    <td></td>
  </tr>
  <tr>
    <td>Q8</td>
    <td><b>D.2.2</b></td> 
    <td><i>TooT-All</i>: Given a fasta file of protein sequences, classify against all transporter schemes.</td>
    <td></td>
  </tr>
  <tr>
    <td>Q8</td>
    <td><b>D.4.2</b></td> 
    <td>Experimental infrastructure - version 2.</td>
    <td><a href="https://github.com/boutiques/boutiques"  target="_BLANK">Link (boutiques)</a><br/>
    <a href="http://www.cbrain.ca/"  target="_BLANK">Link&nbsp;(CBRAIN)</a><br/>
    <a href="https://github.com/bioinformatics-group/bioinformatics-singularity"  target="_BLANK">Link&nbsp;(biolab&nbsp;VM)</a><br/>
    </td>
  </tr>
  <tr>
    <td>Q12</td>
    <td><b>D.1.3</b></td> 
    <td><i>TooT-BERT-ICAT</i>: Utilizes ProtBERT-BFD to predict and classify inorganic ion-specific transmembrane transporter proteins with high precision and accuracy.<br/>
    <i>TooT-SS</i>: Given a fasta file of protein sequences, predict the specific substrate that is transported.</td>
    <td><a href="https://github.com/bioinformatics-group/TooT-BERT_ICAT" target="_BLANK">Link</a></td>
  </tr>
  <tr>
    <td>Q12</td>
    <td><b>D.2.3</b></td> 
    <td><i>TooT-All</i> (version 2): Given a fasta file of protein sequences, classify against all transporter schemes.</td>
    <td></td>
  </tr>
  <tr>
    <td>Q12</td>
    <td><b>D.3.2</b></td> 
    <td><i>TooT-Meta</i> (= <i>TooT-Proteome</i> HPC): Given a fasta file of a meta-proteome, predict the transporters and their properties.</td>
    <td></td>
  </tr>
</table>
</section>

<footer>
Copyright Greg Butler 2019
</footer>

</body>
</html>
